### PR TITLE
Bump lib dependencies

### PR DESCRIPTION
**Pull request by luishfonseca**
# Description

Bumped a bunch of dependencies

## Checklist

- [x] Self-review changes.

### CODE REVIEW

**Comment by RiscadoA** [comment]
Don't .forget to update the [getting started page](https://gamedevtecnico.github.io/cubos/getting-started.html) dependency table.

**Comment by RiscadoA** [comment]
What's the reasoning behind removing the documentation deps?

File: core/CMakeLists.txt [hunk 1]
```diff
@@ -93,8 +93,10 @@ cubos_common_target_options(cubos-core)
 # Link dependencies
 
 if (WITH_OPENGL)
-    add_subdirectory(lib/glad)
-    target_link_libraries(cubos-core PRIVATE glad)
+    set(GLAD_SOUURCES_DIR "lib/glad")
+    add_subdirectory("${GLAD_SOUURCES_DIR}/cmake" glad_cmake SYSTEM)
+    glad_add_library(glad REPRODUCIBLE API gl:core=3.3)
+    target_link_libraries(cubos-core PUBLIC glad)
```
Comment (RiscadoA) [inline core/CMakeLists.txt:99]: Why PUBLIC instead of PRIVATE?

**Review by RiscadoA — CHANGES_REQUESTED** [review]

File: core/CMakeLists.txt (hunk 1 above)
Comment (luishfonseca) [reply core/CMakeLists.txt:99]: Just copied it from the glad documentation, I don't actually know how cmake works

Comment (RiscadoA) [reply core/CMakeLists.txt:99]: By making it PUBLIC you're making dependencies of CUBOS. have it on their include dir too.
Previously it was PRIVATE, try setting it back and see if anything breaks.

**Review by RiscadoA — APPROVED** [review]

**Pull request merged**